server/indexers/transactions.ts

```diff
@@ -12,25 +17,42 @@ export default async function transactions(connection: PoolClient, provider: eth
   async function process() {
     const bids = await findBidsWithMissingTransactions.run({ limit: 5 }, connection);
 
-    log.debug(`Rows: ${bids.length}`);
-
     if (bids.length === 0) {
+      const latestBids = await findWalletsInLatestAuction.run(undefined, connection);
+
+      log.debug(`Rows: ${bids.length} ${latestBids.length}`);
+
+      const latestBalances = await Promise.all(
+        latestBids.map((row) => provider.getBalance(row.walletAddress)),
+      );
```

Comment:
Now that I think about it, Multicall will be helpful here too, there's a get balance call that can be piped to `eth_call`